ml/Host.cc

```diff
@@ -234,6 +234,8 @@ void RrdHost::addDimension(Dimension *D) {
     std::lock_guard<std::mutex> Lock(Mutex);
 
     DimensionsMap[D->getRD()] = D;
+    /*initialize the anomaly counter of the dimension*/
+    D->AnomalousBitCount = 0.0;
```

Comment:
If you default-initialize this, based on the comment I made in `ml/Dimensionh.h`, you can remove this line.